frontend/packages/operator-lifecycle-manager/integration-tests-cypress/tests/operator-install-global.spec.ts

```diff
@@ -127,4 +143,16 @@ xdescribe(`Interacting with a global install mode Operator (${operatorName})`, (
     cy.get('.loading-skeleton--table').should('not.be.visible');
     cy.byTestOperatorRow(operatorRow).should('not.exist');
   });
+
+  it(`deletes the catalogsource`, () => {
```

Comment:
Hi @rhamilto, I know this is a pain but Cypress best practices calls for each `it()` to be independant of each other.  In this case the `create catalogsource` should probably go in a `before()`, and the `delete catalogsource` should go in an `after()`.  -thanks

Reply:
Thanks, @dtaylor113 .  I was seeing some weirdness where both the `before` *and* `after` were running before any of the `it`s.  It appears I have to stop and restart Cypress to resolve.  

Since the `CatalogSource` creation and deletion is duplicated between these two tests, I want to move it to a shared file, but I'm not sure where that should be.  Any advice?

Reply:
Probably need to create new folder/file under `frontend/packages/operator-lifecycle-manager/integration-tests-cypress/views/catalog.ts`